definitions/procedures/installer/upgrade_rake_task.rb

```diff
@@ -2,12 +2,13 @@ module Procedures::Installer
   class UpgradeRakeTask < ForemanMaintain::Procedure
     metadata do
       description 'Execute upgrade:run rake task'
+      confine do
+        feature(:foreman)
```

Comment:
This is run by the installer for Foreman/Katello

Reply:
I don't think this hurts anything and sets up for making a change later.

Reply:
It hurts performance of Foreman/Katello restores where this is now run twice.

https://github.com/theforeman/foreman_maintain/blob/438499be03309045c407c6bce55152c953740c9a/definitions/scenarios/restore.rb#L50

Reply:
Fair, although it's pretty minor:

```
# time foreman-rake upgrade:run
=============================================
Upgrade Step 1/2: katello:correct_repositories. This may take a long while.
=============================================
Upgrade Step 2/2: katello:clean_backend_objects. This may take a long while.
0 orphaned consumer id(s) found in candlepin.
Candlepin orphaned consumers: []

real	0m22.760s
user	0m16.262s
sys	0m6.044s
[root@ip-10-0-168-72 ~]# time foreman-rake upgrade:run
=============================================
Upgrade Step 1/2: katello:correct_repositories. This may take a long while.
=============================================
Upgrade Step 2/2: katello:clean_backend_objects. This may take a long while.
0 orphaned consumer id(s) found in candlepin.
Candlepin orphaned consumers: []

real	0m23.089s
user	0m16.486s
sys	0m6.164s
```

Reply:
I'll remove it for now.

Reply:
I have boxes where it takes 10 minutes 🤷‍♀️